app/assets/javascripts/loginAlert.js

```diff
@@ -3,22 +3,28 @@
 // Set the target date (10 days before March 15th, 2024)
   const targetDate = new Date("April 16, 2024 00:00:00").getTime();
 
+// Set a const for the container to write a conditional statement and only run if it exists
+  const countdownContainer = document.getElementById("countdown-container");
+
  // Function to update the countdown display
   function updateCountdown() {
-    const now = new Date().getTime();
-    const difference = targetDate - now;
-
-    // Time calculations for days only
-    const days = Math.floor(difference / (1000 * 60 * 60 * 24));
-
-    // Visibility logic
-    if (days < 0 || days > 10) {
-      // Hide if more than 10 days away OR if already past the date
-      document.getElementById("countdown-container").style.display = "none";
-    } else {
-      // Show if 10 days or less remaining
-      document.getElementById("countdown-container").style.display = "block";
-      document.getElementById("countdown").innerHTML = days + " days ";
+
+    if (countdownContainer) {
+       const now = new Date().getTime();
```

Comment:
Removing extraneous space:

```suggestion
      const now = new Date().getTime();
```